vagrant/lib/forklift/box_factory.rb

```diff
@@ -29,6 +29,15 @@ def add_boxes!(box_file)
       end
     end
 
+    def filter_boxes!
+      box_config = Settings.new.settings['boxes']
+      return unless box_config&.key?('exclude')
```

Comment:
Not seen that syntax before, what does it mean?

Reply:
Don't fail if `box_config` is `nil` and thus has no `key?`

http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/

Reply:
This is Ruby 2.4+ I believe and unsupported on EL7, right?

Reply:
2.3+ I think, but yes.

And before you ask: Vagrant on EL7 uses Ruby 2.4 that it bundles:

```console
# /opt/vagrant/embedded/bin/ruby --version
ruby 2.4.9p362 (2019-10-02 revision 67824) [x86_64-linux]
```

Reply:
That would be problematic for at least one of my setups. Let's use the old school method :)

Reply:
@evgeni https://github.com/theforeman/forklift/blob/master/bin/ansible-vars would break

Reply:
Ewwww. OK